src/python/ensembl/io/genomio/genome_metadata/extend.py

```diff
@@ -57,81 +46,74 @@ def get_additions(report_path: Path, gbff_path: Optional[Path]) -> List[str]:
     """
     gbff_regions = set(get_gbff_regions(gbff_path))
     report_regions = get_report_regions_names(report_path)
-
     additions = []
-    for rep_seq in report_regions:
-        (rs_seq, gb_seq) = rep_seq
-        if rs_seq not in gbff_regions and gb_seq not in gbff_regions:
-            if rs_seq:
-                additions.append(rs_seq)
+    for seq_region_name in report_regions:
+        (genbank_seq_name, refseq_seq_name) = seq_region_name
+        if genbank_seq_name not in gbff_regions and refseq_seq_name not in gbff_regions:
+            if refseq_seq_name:
+                additions.append(refseq_seq_name)
             else:
-                additions.append(gb_seq)
+                additions.append(genbank_seq_name)
     additions = sorted(additions)
     return additions
 
 
-def get_gbff_regions(gbff_path: Optional[Path]) -> List[str]:
-    """Returns the `seq_region` data from the GBFF file.
+def get_gbff_regions(gbff_path: Optional[PathLike]) -> List[str]:
+    """Returns the `seq_region` data from a GBFF file.
 
     Args:
-        gbff_path: Gbff file path to use.
+        gbff_path: GBFF file path to use.
     """
-    if not gbff_path:
-        return []
-
     seq_regions = []
-    with open_gz_file(gbff_path) as gbff_file:
-        for record in SeqIO.parse(gbff_file, "genbank"):
-            record_id = re.sub(_VERSION_END, "", record.id)
-            seq_regions.append(record_id)
+    if gbff_path:
+        with open_gz_file(gbff_path) as gbff_file:
+            for record in SeqIO.parse(gbff_file, "genbank"):
+                record_id = re.sub(_VERSION_END, "", record.id)
+                seq_regions.append(record_id)
     return seq_regions
 
 
-def _report_to_csv(report_path: Path) -> Tuple[str, dict]:
-    """Returns an assembly report as a CSV string, and the head metadata as a dict.
+def _report_to_csv(report_path: PathLike) -> Tuple[str, Dict]:
+    """Returns the assembly report as a CSV string, and its metadata as a dictionary.
 
     Args:
-        report_path: Path to a `seq_region` file from INSDC/RefSeq.
-
+        report_path: Path to the assembly report file from INSDC/RefSeq.
     """
     data = ""
     metadata = {}
-    with report_path.open("r") as report:
-        last_head = ""
+    with Path(report_path).open("r") as report:
+        prev_line = ""
         for line in report:
-            # Ignore header
             if line.startswith("#"):
                 # Get metadata values if possible
                 match = re.search("# (.+?): (.+?)$", line)
```

Comment:
mb use something like `#\s*(.+?):\s+(.+?)\s*$` or `#\s*([^:]+?):\s+(.+?)\s*$`

Reply:
That would be safer indeed and more convenient, cheers!